apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -29,9 +38,20 @@ const FieldText = styled(Text).attrs(() => ({
   max-width: 50%;
 `;
 
+const HorizontalSeparator = styled.div`
+  height: 1px;
+  background: ${p => p.theme.colors.palette.text.shade20};
+  width: 100%;
+`;
+
 export type FieldComponentProps = FCPGeneric<Account, TransactionCommon, TransactionStatus>;
 export type FieldComponent = React.ComponentType<FieldComponentProps>;
 
+const termsOfUse = {
+  paraswap: "https://paraswap.io/tos",
+  "1inch": "https://1inch.io/assets/1inch_network_terms_of_use.pdf",
+};
```

Comment:
Using `as const` here should allow you to avoid the `as` keyword later on to access by key
```suggestion
const termsOfUse = {
  paraswap: "https://paraswap.io/tos",
  "1inch": "https://1inch.io/assets/1inch_network_terms_of_use.pdf",
} as const;
```

Reply:
TS still screaming at me

Reply:
[ref](https://github.com/LedgerHQ/ledger-live/pull/6174#discussion_r1490692545)